### PR TITLE
DNS form improvements

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/dns/dnsaas.py [hunk 1]
```diff
@@ -78,7 +77,7 @@ def update_dns_records(self, record):
             record: record cleaned data
 
         Returns:
-            True or False if an error from api
+            Validation error from API or None if update corrent
```
Comment (ghost) [inline src/ralph/dns/dnsaas.py]: `corrent` -> `correct`?

File: src/ralph/dns/dnsaas.py [hunk 2]
```diff
@@ -126,7 +125,7 @@ def create_dns_records(self, record):
             records: Record cleaned data
 
         Returns:
-            True or False if an error from api
+            Validation error from API or None if create corrent
```
Comment (ghost) [inline src/ralph/dns/dnsaas.py]: `corrent` -> `correct`?

File: src/ralph/dns/dnsaas.py [hunk 3]
```diff
@@ -162,7 +161,7 @@ def delete_dns_records(self, record_id):
             record_ids: ID's to delete
 
         Returns:
-            True or False if an error from api
+            Validation error from API or None if delete corrent
```
Comment (ghost) [inline src/ralph/dns/dnsaas.py]: `corrent` -> `correct`?

File: src/ralph/dns/views.py [hunk 4]
```diff
@@ -35,12 +35,14 @@ def get_forms(self):
...
     def get(self, request, *args, **kwargs):
-        kwargs['forms'] = self.get_forms()
+        if 'forms' not in kwargs:
+            kwargs['forms'] = self.get_forms()
         return super().get(request, *kwargs, **kwargs)
 
     def post(self, request, *args, **kwargs):
         forms = self.get_forms()
         posted_form = DNSRecordForm(request.POST)
+        # Looking for a form that is create new record
```
Comment (ghost) [inline src/ralph/dns/views.py]: To me this is more like:
`# find form which request's data belongs to`

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**